snap/snapcraft.yaml

```diff
@@ -152,6 +165,17 @@ parts:
     - wayland
     - nvidia
     plugin: nil
+    build-snaps:
+    - core24
+    override-prime: |
+      set -eux
+      cd /snap/core24/current
+      find . -type f,l -exec rm -f ${CRAFT_PRIME}/{} \;
```

Comment:
`find . -type f,l -exec rm -f ${CRAFT_PRIME}/{} \;` this fails with `cannot remove '/nvidia/nvidia-2024/prime/./bin': Is a directory` but the snap still builds. Is this concerning?

Reply:
No, should be fine. It's just that `/bin` is a symlink to `/usr/bin` in the `core24` snap, and not in the snap you're building.

You can check what ends up in `/bin` in the resulting snap.